### PR TITLE
debug window enhancements

### PR DESCRIPTION
**Pull request by ZappoMan**
- adds a menu item to the developer menu to easily launch this script
- adds date/time to the window's log text

### CODE REVIEW

**Comment by ZappoMan** [comment]
Test plan:
- notice that the Developer menu now has a `Script Log (HMD friendly)...` menu option and it runs the Debug Window script
- notice that the window log now has time stamps

**Comment by hifi-gustavo** [comment]
Build console output for commit: e6a20102d4653e17ad9b6383a31bc6253dc69007 available at: https://gist.github.com/b062a848d8451b64db994d1b284cbba5<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9251-e6a2010.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9251-e6a2010.dmg

**Comment by OmegaHeron** [comment]
Tested Windows 10
QA Pass
General smoke test = pass.
Displays proper in desktop and HMD (DK2) modes.

[11/21 15:6:9] [] Fooooo  (comes from me issuing a print("Fooooo"); in js console)
Timestamps could use some leading zero formatting....

Desktop

hifi-snap-by-omegaheron-on-2016-12-21_15-08-43

HMD

hifi-snap-by-omegaheron-on-2016-12-21_15-03-00

File: scripts/developer/debugging/debugWindow.js [hunk 1]
```diff
@@ -19,18 +20,33 @@ var window = new OverlayWindow({
 window.setPosition(25, 50);
 window.closed.connect(function() { Script.stop(); });
 
+var getFormattedDate = function() {
+    var date = new Date();
+    return date.getMonth() + "/" + date.getDate() + " " + date.getHours() + ":" + date.getMinutes() + ":" + date.getSeconds();
```
Comment (davidkelly) [inline scripts/developer/debugging/debugWindow.js]: maybe leading 0s for month, day, hour, min and sec less than 10?

**Comment by davidkelly** [comment]
Marked dev approved -- though leading 0s (at least for minutes and seconds?) seems right.

**Pull request merged**